README.md

```diff
@@ -41,4 +41,13 @@ Code of Conduct
 
 We have our own [Code of Conduct](Code of Conduct.md), which is adapted from the [Contributor Covenant](http://contributor-covenant.org), version 1.3.0, available at [http://contributor-covenant.org/version/1/3/0/](http://contributor-covenant.org/version/1/3/0/). The CoC is taken seriously by the project owners.
 
+Automatic PR checking with Peril
+--------------------------------
 
+[Peril](https://github.com/Danger/Peril) is a server which runs [Danger](http://danger.systems/js/) automatically, on all pull requests, organization-wide. We can check for pull request metadata, commit information, which files were changed, all kinds of thigs. Here is the list of checks which currently get run:
```

Comment:
A few minor notes here:

- You wrote "thigs" instead of "things"
- Can we be specific about the fact this supports only Danger.JS scripts ?
- I think "We can check" is better off as "It can check" (relating to Peril and not the community) 
- For the last sentence what do you think of "Here are some of the things Peril checks on *every* RxSwiftCommunity repo:"

Let me know your thoughts 😄

Reply:
Sounds good, I'll push those changes momentarily. I'd like to keep the list in the readme exhaustive though, since it's a lot easier than parsing through all the code :)